src/controllers/index.controller.js

```diff
@@ -7,8 +7,7 @@
 
 const { walletApi, tokensUtils, walletUtils, Connection, HathorWallet, Network, helpersUtils, SendTransaction, constants: hathorLibConstants } = require('@hathor/wallet-lib');
```

Comment:
It's funny that the linter didn't complain about importing `Connection` and `HathorWallet`. They are not used anymore in this file.

Reply:
Indeed! Fixed on [fa1806e](https://github.com/HathorNetwork/hathor-wallet-headless/pull/392/commits/fa1806e17151b9674af208b25931b005604bb8ca).